.github/workflows/amzn1.yml

```
@@ -101,5 +101,5 @@ jobs:
           git config --local user.name "Bryce Kahle"
           git config --local user.email "bryce.kahle@datadoghq.com"
           git add -A
-          git diff-index --quiet HEAD || git commit -m "amzn 1 update on $(date -u +%Y-%m-%d)"
+          git diff-index --quiet HEAD || (git commit -m "amzn 1 update on $(date -u +%Y-%m-%d)" && git pull --rebase)
           git push
```

Maybe it's an unlikely scenario where two workflows do the pull at the same time and then one of the pushes fails, but I think it would be better to do the rebase if the push fails, something like this

```suggestion
          git diff-index --quiet HEAD || git commit -m "amzn 1 update on $(date -u +%Y-%m-%d)"
          git push || (git pull --rebase && git push)
```